### DISCUSSION TITLE
Loader

### DISCUSSION THREAD

**Discussion by badminimum** [solidjs/solid-router#235] [Q&A]
How can I show an specific component on every site load?

Like a Loader Element

**Comment by Brendan-csel** [comment]
So not route specific, and shows for all routes?  In that case put it outside your Routes/Route components.

**Reply by badminimum** [reply]
I mean how can I show it on every site?

**Reply by Brendan-csel** [reply]
I'm not sure I understand.  I guess you'd just use your component in every site.

**Reply by badminimum** [reply]
I already thought about that to, but I don't want that the component need to load like the site so it can instantly pop up
